rulebook.tex

```diff
@@ -380,193 +435,196 @@ \subsection{Executive Committee 2015--2019}
-\reffig{fig:competition-area}. Machines have two kind of sides: active
-(wide, where the robot interacts) and ignored (narrow) edges.  The
-constraints for positioning of the MPS are given in
-\refsec{sec:machine-swapping}.
-
-The distribution and alignment of all machines is axially symmetrical
-to the y-axis. Thus, each team has similar conditions on both halves of
-the competition field. Both teams have an exclusive set of 7 out of
-the 14 machines each. Machines from one team can also be located on the
-primary side of the opposing team. We will later discuss the concept
-of the production machine distribution for each team in detail.
-
-The center coordinates and the size of the zones in which the machines
-are placed can be seen in \reffig{fig:competition-area}. The zone
-names are structured as a team prefix (``C-'' for cyan, ``M-'' for
-magenta), Z to distinguish zone names, and a grid coordinate. For
-example, The zone C-Z23 is the zone on the cyan primary half, which is
-the second along the X-axis and the third along the Y-axis. Zones are
-only a means to position machines. They will not be physically
-represented or visible on the field. They cannot be used for any other
-purpose.
-
-Note that this specific field layout is an example only, accounting
-for symmetry and avoiding clustering of machine access nodes and
-unapproachable machines. However, the actual distribution and
-alignment of machines on the competition area will change before the
-actual game starts and will be different for each game. Thus, teams
+The competition area of the \ac{RCLL} has a rectangular shape. The field is
+divided in square zones of \SI{1 x 1}{\metre}. %chktex 29
+The field is partially surrounded (at least 50\% but not more than 70\%)
+by wall elements of at least \SI{0.5}{\metre} height.
+Different machines are placed within the area, the constraints for
+positioning them are given in \refsec{sec:machine-swapping}.
+The \ac{RCLL} supports two kind of field layouts:
+Firstly, a symmetrical one
+with aspect ratio 1:2 intended for two teams to compete simultaneously
+on the same area.
+Secondly, a field with a squared layout that is intended for tasks,
+where only one team is on the field at a time.
+Fields also contain designated robot insertion areas, that are used as a
+starting position for the robots when a game starts or a robot is inserted
+into a running game. The usage of these areas is restricted to these purposes,
+unless stated otherwise.
+In the following, the different field layouts are presented.
+The depicted field layouts are examples only, the actual
+distribution and alignment of machines on the competition area will change
+before the actual game starts and will be different for each game. Thus, teams
 should focus on a generic approach for production, allowing for
 dynamic adaptation of machine positions and alignments.
 
-\subsubsection{Moving on the Field}
+\subsection{Fields with Aspect Ratio 2}
```

Comment:
I would call it Main Track field

Reply:
I kept most of the rulebook independent from the played format to not switch around between the differences too much, thats why i kept the lame names. I added your proposed names to the subsection name as a compromise as i think in this case it does make sense to have a name for the different fields.